cardinal/server/server.go

```diff
@@ -35,10 +28,11 @@ var (
 )
 
 const (
-	listTxEndpoint   = "/list/tx-endpoints"
-	listReadEndpoint = "/list/read-endpoints"
+	listTxEndpoint   = "list/tx-endpoints"
+	listReadEndpoint = "list/read-endpoints"
+	listTxReceipts   = "list/tx-receipts"
```

Comment:
hm, i think we shouldn't use `list/tx-receipt`. we should be consistent of using `list/` prefix only to list endpoints with a certain category. 

i think it would be great to consolidate the endpoints convention to something like this

`list/tx` -> list tx endpoints
`list/read` -> list read endpoints
`/tx/module/abc` -> a tx endpoint (i.e. `/tx/persona/abc`)
`/read/module/abc` -> a read endpoint (i.e. `/read/receipt/abc`)

Reply:
I renamed the tx receipts endpoint so it isn't prefixed with list, nor is it prefixed with "tx-" (making ti look like a normal, user-defined transaction). 

I don't know about having the module as part of the path. I agree in principle, however I think getting the module name is going to be messy.

Reply:
i dont mean `module` as go module. it's basically just arbitrarily set categories

Reply:
Ah, in that case, there's nothing stopping a user from making whatever groupings they want. The only piece we need to do would be to replace tx- with tx/. Are you suggesting that for this PR?

Reply:
what i want to have

`tx/persona/abc` -> all persona tx is like this
`tx/game/abc` -> all tx that is defined by game implementation goes here

same with read

`read/persona/abc` -> all persona read is like this
`read/receipt/abc` -> receipts read
`read/game/abc` -> all read that is defined by game implementation goes here

i want it to be very explicit from the endpoint on what functionality/feature/category/etc this `tx` or `read` belongs to